release notes for 1.8.2 (#411)
okteto/docs: versioned_docs/version-1.8/self-hosted/install/releases.mdx

id: releases
---

## 1.8.2
26 May, 2023

### Bugfixes

> - Upgraded the Okteto CLI version to [2.15.3](https://github.com/okteto/okteto/releases/tag/2.15.3).

Reviewer: ```suggestion
- Upgraded the Okteto CLI version to [2.15.4](https://github.com/okteto/okteto/releases/tag/2.15.4).
```

>   - Build optimization now considers environment variables used in build args in map format to determine when to rebuild an image.

Reviewer 2: ```suggestion
- Build optimization now also considers the environment variables used in build args when determining when to rebuild an image.
```
